architecture/core_concepts/routes.adoc

```diff
@@ -302,6 +303,27 @@ ports that the router is listening on, `ROUTER_SERVICE_SNI_PORT` and
 they are unique on the machine. These ports will not be exposed externally.
 ====
 
+[[strict-sni]]
+== HAProxy Strict SNI
+
+By default, when a host does not resolve to a route in a HTTPS or TLS
+SNI request, the default certificate is returned to the caller as part of the
+503 response. This exposes the default certificate and can pose security
+concerns. HAProxy `strict-sni` option to bind suppresses use of the
```

Comment:
"can pose security concerns because the wrong certificate is served for a site."